include/cpuinfo_ppc.h

```diff
@@ -75,11 +74,12 @@ typedef struct {
 PPCInfo GetPPCInfo(void);
 
 typedef struct {
-  char platform[64];  // 0 terminated string
-  char model[64];     // 0 terminated string
-  char machine[64];   // 0 terminated string
-  char cpu[64];       // 0 terminated string
-  PlatformType type;
+  char platform[64];       // 0 terminated string
+  char model[64];          // 0 terminated string
+  char machine[64];        // 0 terminated string
+  char cpu[64];            // 0 terminated string
+  char platform[64];       // 0 terminated string
```

Comment:
two platform field -> need to rename one of them:
note: both won't contains the same value (according to your mock test)

Reply:
Yes that's why I intend to roll back as I said in https://github.com/google/cpu_features/pull/164#discussion_r658625449. It also breaks client code so not ideal.

Reply:
done